fr/lecons/nettoyer-ses-donnees-avec-openrefine.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Nettoyer ses données avec OpenRefine
+layout: lesson
+slug: nettoyer-ses-donnees-avec-openrefine
+date: 2013-08-05
+translation_date: 2018-11-27
```

Comment:
Sorry, one more note - please update this date to `2019-04-10` so it's closer to the publication date